LayoutTests/accessibility/element-path-with-css-transforms.html

```
@@ -0,0 +1,85 @@
+    margin: -40px;
+    border-radius: 50%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+/* Three items at 120-degree intervals around a circle. */
+#item1 { transform: rotate(0deg) translateX(-120px) rotate(0deg); }
+#item2 { transform: rotate(120deg) translateX(-120px) rotate(-120deg); }
+#item3 { transform: rotate(240deg) translateX(-120px) rotate(-240deg); }
+</style>
+</head>
```

HTMLHint (1.9.2)

[error] <title> must be present in <head> tag.

(title-require)